### PR TITLE
DNM - Migrate modules ec2_vpc_egress_igw and tests

### PR DESCRIPTION
**Pull request by abikouo**
- Migrage modules ec2_vpc_egress_igw and tests from collection community.aws

### CODE REVIEW

**Review by GomathiselviS — APPROVED** [review]

File: plugins/modules/ec2_vpc_egress_igw.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    vpc_id: vpc-abcdefgh
+    state: absent
+"""
+
+RETURN = r"""
+gateway_id:
+    description: The ID of the Egress Only Internet Gateway or Null.
+    returned: always
+    type: str
+    sample: eigw-0e00cf111ba5bc11e
```
Comment (alinabuzachis) [inline plugins/modules/ec2_vpc_egress_igw.py:59]:
```suggestion
    sample: "eigw-0e00cf111ba5bc11e"
```

File: plugins/modules/ec2_vpc_egress_igw.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+vpc_id:
+    description: The ID of the VPC to attach or detach gateway from.
+    returned: always
+    type: str
+    sample: vpc-012345678
```
Comment (alinabuzachis) [inline plugins/modules/ec2_vpc_egress_igw.py:64]:
```suggestion
    sample: "vpc-012345678"
```

File: plugins/modules/ec2_vpc_egress_igw_info.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+---
+module: ec2_vpc_egress_igw_info
```
Comment (alinabuzachis) [inline plugins/modules/ec2_vpc_egress_igw_info.py:9]: I'm not sure if this module should be in the same PR. I guess we should only include the modules we'd want to migrate from c.aws to a.aws.

Comment (alinabuzachis) [reply plugins/modules/ec2_vpc_egress_igw_info.py:9]: I'd suggest moving this module into a separate PR. We'll merge it first so there would not be any issue with the integration tests.

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**